docs/user-guide/resource-browser.md

```diff
@@ -1,71 +1,438 @@
+
+![Figure 12a: Visual Representation - Draining a Node](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/drain-visual.jpg)
+
+After the node is drained, all pods (including those managed by DaemonSets) in the node will be automatically drained to other nodes in the cluster, and the drained node will be set to cordoned status.
+
+* Click **Drain**.
+
+  ![Figure 12b: Draining a Node](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/drain.jpg)
+
+* A confirmation dialog box will appear, click **Drain Node** to proceed.
+
+  ![Figure 12c: Drain Confirmation](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/drain-dialog.jpg)
+
+You can also select from the following conditions before draining a node:
+
+| Name | Usage |
+| --- | --- |
+| Grace Period | Period of time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used. |
+| Delete empty directory data | Enabling this field will delete the pods using empty directory data when the node is drained. |
+| Disable eviction (use with caution) | Enabling this field will force drain to use delete, even if eviction is supported. This will bypass checking `PodDisruptionBudgets`.<br>Note: Make sure to use with caution.</br> |
+| Force drain | Enabling this field will force drain a node even if there are pods that do not declare a controller. |
+| Ignore DaemonSets | Enabling this field will ignore DaemonSet-managed pods. |
+
+### Taint a Node
+
+Taints are `key:value` pairs associated with effect. After you add taints to nodes, you can set tolerations on a pod to allow the pod to be scheduled to nodes with certain taints. When you taint a node, it will repel all the pods except those that have a toleration for that taint. A node can have one or many taints associated with it.
+
+![Figure 13a: Visual Representation - Tainting a Node](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/taint-visual.jpg)
+
+**Note**: Make sure to check taint validations before you add a taint.
+
+* Click **Edit taints**.
+
+  ![Figure 13b: Tainting a Node](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/edit-taints.jpg)
+
+* Enter the `key:value` pairs and select the [taint effect](#taint-effects) from the drop-down list.
+
+  ![Figure 13c: Adding Taints](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/taint-dialog.jpg)
+
+* Click **Save**.
+
+You can also add more taints using **+ Add taint button**, or delete the existing taint by using the delete icon.
+
+#### Taint Effects
+
+A taint can produce three possible effects:
```

Comment:
We should ideally link such conceptual things to k8s official docs. and in our doc we should focus on how to's...